content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -182,20 +180,15 @@ template, the StatefulSet starts to create new Pods based on the updated templat
 Eventually, all of the old Pods are replaced with new Pods, and the update is complete.
 
 Each workload resource implements its own rules for handling changes to the Pod template.
-If you want to read more about StatefulSet specifically, read
-[Update strategy](/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets) in the StatefulSet Basics tutorial.
+To learn how StatefulSet resources manage Pod updates, read the
+[StatefulSet Basics tutorial](/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets).
 
-On Nodes, the {{< glossary_tooltip term_id="kubelet" text="kubelet" >}} does not
+On nodes, the {{< glossary_tooltip term_id="kubelet" text="kubelet" >}} does not
 directly observe or manage any of the details around pod templates and updates; those
 details are abstracted away. That abstraction and separation of concerns simplifies
 system semantics, and makes it feasible to extend the cluster's behavior without
 changing existing code.
 
-## Resource sharing and communication
-
-Pods enable data sharing and communication among their constituent
-containers.
-
```

Comment:
If we remove the  “Resource sharing and communication” heading, then should this become a 2nd level heading, along with Pod networking? Otherwise, these become grouped under “Working with Pods”, which doesn't seem right.

Two more options:
- move “Storage in Pods” and “Pod networking” to come immediately after “How Pods manage multiple containers”
- move “How Pods manage multiple containers” down to here.

Reply:
Thanks for reviewing. Will look at this again.

Reply:
I think it makes sense to follow the "Using Pods" main section with the 
"How Pods manage multiple containers", Storage, and Networking sections.
Let me know if you have other concerns.